README.md

```diff
@@ -1 +1,68 @@
-# sdx-encrypted-submitter
\ No newline at end of file
+# sdx-encrypted-submitter
+
+A command line utility to copy a files contents to a rabbit exchange
+
+Building and Running
+
+Install Go and ensure that your GOPATH env is set ( usually it's ~/go).
+
+```
+go get  github.com/ONSdigital/sdx-encrypted_submitter/
+cd $GOPATH/src/sdx-encrypted-submitter
+go get -u github.com/streadway/amqp
+go get -u gopkg.in/yaml.v2
+go build
+```
+
+### Notes: 
+* govender support likely to follow. (https://github.com/kardianos/govendor)
+* No unit tests yet
+* Assumes exchanges , queues and bindings are in place . Does NOT create them if they are not.
+
+
+### Useage
+
+Arguments for use are split between sdx-encrypted-submitter.yml and command line arguments
+
+#### Config file settings (sdx-encrypted-submitter.yml) 
+
+Name | Default
```

Comment:
Shouldn't this be Name | Default | Description? as the second column just describes what the setting is used for.  Also need to add any defaults (if any, i cant remember)